src/complex_tensor/test/test_ops.py

```diff
@@ -62,6 +66,24 @@ def matches(self, other: TestInfo) -> bool:
     TestInfo(
         op_name="sub", dtype=torch.complex32, compile=True
     ): "numerical precision optimized out",
+    TestInfo(
+        op_name="sort"
+    ): "https://github.com/pytorch/pytorch/pull/159556#issuecomment-3154215299",
+    TestInfo(
+        op_name="minimum"
+    ): "https://github.com/pytorch/pytorch/pull/159556#issuecomment-3154215299",
+    TestInfo(
+        op_name="maximum"
+    ): "https://github.com/pytorch/pytorch/pull/159556#issuecomment-3154215299",
+    TestInfo(
+        op_name="argmin"
+    ): "https://github.com/pytorch/pytorch/pull/159556#issuecomment-3154215299",
+    TestInfo(
+        op_name="argmax"
+    ): "https://github.com/pytorch/pytorch/pull/159556#issuecomment-3154215299",
+    TestInfo(
+        op_name="topk"
+    ): "https://github.com/pytorch/pytorch/pull/159556#issuecomment-3154215299",
```

Comment:
These should be removed when https://github.com/pytorch/pytorch/pull/159556#issuecomment-3154215299 is resolved.

Reply:
Created an issue: https://github.com/openteams-ai/pytorch-complex-tensor/issues/8